src/components/RoiCalculator.vue

```diff
@@ -0,0 +1,432 @@
+    >
+      <div
+        v-bem="'content'"
+      >
+        <span v-bem="'label'">
+          Staked for
+        </span>
+        <KlayTabs
+          v-model="stakeFor"
+          :tabs="stakeTabs"
+        />
+        <KlaySwitch
+          v-model="compoundingEnabled"
+          v-bem="'compounding-checkbox'"
+          label="Compounding every"
+        />
+        <KlayTabs
+          v-model="compoundingEvery"
+          :tabs="compoundingTabs"
+          :disabled="!compoundingEnabled"
+        />
+        <hr>
+        <span v-bem="'label'">
+          Amount staked
+        </span>
+        <div
+          ref="stakeValueInputWrapper"
+          v-bem="'input-wrapper'"
+        >
+          <KlayTextField
+            v-bem="'input'"
+            :model-value="stakeValueRaw"
+            @update:model-value="(value: string) => updateStakeValueRaw(value, true)"
+          />
+          <KlayIconSwitch
+            v-bem="'input-switch'"
+            @click="switchUnits"
+          />
+          <div
+            v-bem="'input-another-units'"
+          >
+            {{ formattedStakeValueInAnotherUnits }}
+          </div>
+        </div>
+        <div v-bem="'amounts'">
+          <KlayButton
+            v-for="amount in [10, 100, 1000]"
+            :key="amount"
+            v-bem="'amount'"
+            size="sm"
+            @click="setStakeValueInUSD(amount)"
+          >
+            ${{ amount }}
+          </KlayButton>
+          <KlayButton
+            v-bem="'amount'"
+            size="sm"
+            @click="setStakeValueWithBalance()"
+          >
+            My balance
+          </KlayButton>
+        </div>
+        <span v-bem="'label'">
+          You will receive (APY = {{ apy.toFixed(2) }}%)
+        </span>
+        <div v-bem="'input-wrapper'">
+          <KlayTextField
+            v-model="formattedReceiveValue"
+            v-bem="'input'"
+            :disabled="true"
+          />
+          <div
+            v-bem="'input-another-units'"
+          >
+            {{ formattedReceiveValueInAnotherUnits }}
+          </div>
+        </div>
+      </div>
+      <KlayCollapse v-bem="'details'">
+        <template #head>
+          Details
+        </template>
+        <template #main>
+          <div
+            v-for="item in detailsList"
+            :key="item.label"
+            v-bem="'details-item'"
+          >
+            <div v-bem="'details-item-label'">
+              {{ item.label }}
+            </div>
+            <div v-bem="'details-item-value'">
+              {{ item.value }}
+            </div>
+          </div>
+          <div v-bem="'details-description'">
+            {{ detailsDescription }}
+          </div>
+        </template>
+      </KlayCollapse>
+    </KlayModalCard>
+  </SModal>
+</template>
+
+<style lang="sass">
+@import '@/styles/vars.sass'
+
+.roi-calculator
+  width: 345px
+  hr
+    width: 100%
+    height: 1px
+    margin: 16px 0
+    background-color: $gray5
```

Comment:
You can use `.klay-divider` class for it, defined at `@/styles/main.sass` (if I remember correctly).

Reply:
Done